device_attributes/index.bs

```diff
@@ -0,0 +1,192 @@
+ 
+With the help of Device Attributes API, this sales application can get the device location by using {{getAnnotatedLocation()}} method. The operation will fail if the API call is triggered by a phishing application - looks similar but not the expected one.
+ 
+<div class="example">
+ 
+<xmp class="lang-js">
+// Request sensitive data if the current environment is valid.
+function successCallback(location) {
+  const tariff = backend.requestTariff(location);
+  console.log(tariff);
+}
+ 
+// Stop the workflow if the current environment is unexpected.
+function failureCallback(error) {
+  backend.reportFailure(error);
+  console.error(error.message);
+}
+ 
+function PrepareTariff() {
+  navigator.managed.getAnnotatedLocation()
+                   .then(successCallback, failureCallback);
+}
+</xmp>
+ 
+</div>
+ 
+It is easy to write another similar code snippet to report the sales data including a device serial number by using {{getAnnotatedAssetId()}} method. 
+ 
+NOTE: The website should assume that the data could be tampered with, stolen or reused on another managed device. Alternative security measures should be put in place by the website itself.)
+ 
+<div class="example">
+ 
+<xmp class="lang-js">
+function ReportSalesData() {
+  navigator.managed.getAnnotatedAssetId.then(reportCallback);
+}
+</xmp>
+ 
+</div>
+ 
+# Security considerations # {#security-considerations}
+ 
+In accordance with modern security practices, the permission of using these methods is controlled by the administrator per origin. In addition, they are only available to [=secure contexts=] on the managed devices.
+
+# Privacy considerations # {#privacy-considerations}
+
+<pre class=biblio>
+{
+  "MANAGED-CONFIG": {
+    "authors": [
+      "Anatoliy Potapchuk"
+    ],
+    "href": "https://wicg.github.io/WebApiDevice/managed_config",
+    "title": "MANAGED-CONFIG"
+  }
+}
+</pre>
+ 
+This API provides access to device attributes which will be the same for all origins. (See [[MANAGED-CONFIG]] for per-origin configuration of managed sites.) This could easily be used for tracking purposes if exposed arbitrarily. In a managed environment however, where an administrator is configuring a fleet of machines for particular employees or tasks, it is desirable for these machines to be able to identify themselves to the sites they are expected to be accessing.
+ 
+It is a non-goal for this interface to be used on unmanaged devices. An end user must never be asked to provide this information to a site. The decision of which origins are granted permission to access these device attributes must be made by an authorized administrator of the organization to which they belong. Most operating systems have a mechanism for the deployment of such policies, which can be used to configure many aspects of a device's operating system or applications such as the web browser. This specification requires that permission to access these device attributes is configured through such a system.
```

Comment:
You should give this background information ("Most operating systems have ...") somewhere outside of the privacy considerations section, since it directly affects how UAs have to implement this API. Then you can refer to how it's used to enforce privacy properties here.

Reply:
Try to avoid "must" statements inside security and privacy considerations sections. Any normative requirements (which "must" introduces) should be somewhere else in the specification, and then the privacy considerations section can describe the privacy goals they achieve, and refer to them as simple facts like "End users are never asked to provide this information to a site ([[#section-with-the-normative-requirement]])".

Reply:
I moved "Most operating systems have ..." and some other description to the new 'Permission Control'. All 'must's have been removed as suggested.